source/postprocess/visualization/heating.cc

```diff
@@ -38,7 +38,8 @@ namespace aspect
       Heating<dim>::
       Heating ()
         :
-        DataPostprocessor<dim> ()
+        DataPostprocessor<dim> (),
+        Interface<dim>("units")
```

Comment:
I think this should be as follows:
```suggestion
        Interface<dim>("W/kg")
```